.github/workflows/build_website_preview.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy Vector Website Preview Site
```

Comment:
Not a big deal, but given these workflows are almost identical, we could create just one parameterized workflow and call it from each of these site-specific workflows. See https://docs.github.com/en/actions/using-workflows/reusing-workflows